app/src/main/java/com/asfoundation/wallet/billing/partners/OemIdExtractorService.kt

```diff
@@ -11,6 +11,7 @@ class OemIdExtractorService(
 
   fun extractOemId(packageName: String): Single<String> {
     return extractorV2.extract(packageName)
+        .doOnSuccess { extracted -> if (extracted.isEmpty()) throw IllegalStateException() }
```

Comment:
Replace "if" for "check(extracted.isNotEmpty())"

Reply:
done